frontend/components/dist/components.min.js.LICENSE.txt

```diff
@@ -0,0 +1,24 @@
+/*!
```

Comment:
not too sure what triggered the separation (might have been the bundle name change), but it makes the bundle smaller anyway 😀